### PR TITLE
Update cftime gsw

### PR DESCRIPTION
**Pull request by ocefpaf**
### Description

Updating gsw and cftime libraries.

### Checklists

- [X] Add a [CHANGELOG](https://github.com/pyodide/pyodide/blob/main/docs/project/changelog.md) entry
- [X] Add / update tests
- [X] Add new / update outdated documentation

### CODE REVIEW

**Comment by ocefpaf** [comment]
Is this the correct way to update existing libraries? I wonder if they will get stale quickly without a community of maintainers or some sort of automatic updating system. We (conda-forge) can probably help with the latter.

**Review by ryanking13 — COMMENTED** [review]
Thanks, @ocefpaf!

> Is this the correct way to update existing libraries? I wonder if they will get stale quickly without a community of maintainers or some sort of automatic updating system. We (conda-forge) can probably help with the latter.

Yes, this is the correct way and I agree that it gets stale quickly (we normally try to update all packages before release). Currently, we lack an automatic updating system and it would be really helpful if `conda-forge` team can contribute to this!

File: docs/project/changelog.md [hunk 1]
```diff
@@ -159,6 +159,9 @@ substitutions:
 - {{ Update }} Upgraded pandas to version 1.5.0.
   {pr}`3134`
 
+- {{ Update }} Upgraded gsw and cftime to versions 3.6.16 and 1.6.2 respectively.
```
[the remark this answers is not in the document]
Comment (ocefpaf) [reply docs/project/changelog.md:162]: Makes sense. It is too noisy. I'll remove that tomorrow morning.

[the remark this answers is not in the document]
Comment (ocefpaf) [reply docs/project/changelog.md]: Done!

**Comment by ocefpaf** [comment]
> Yes, this is the correct way and I agree that it gets stale quickly (we normally try to update all packages before release). Currently, we lack an automatic updating system and it would be really helpful if `conda-forge` team can contribute to this!

Two quick questions:

- An update in topological order doesn't make sense here b/c it is a monorepo and everything gets build, right? (If so this process would be as simple as a script that updates all in a single run.)
- An update script that runs once per pyodide release may be easier but it would add an extra burden on maintainers, who would have to scramble to fix whatever is broken. While constant updates to every new package release would reduce that burden, it may put some strain on the CIs here. How about something that runs, say, weekly?

If my guesses are correct your needs here would be simpler and easier to manage then conda-forge's. A update script that would be in a GHA cron job that runs weekly and send a PR updating all packages if updates are available.

**Comment by hoodmane** [comment]
I think your analysis is exactly right @ocefpaf and that sounds great. As a later thing, to have some way of tracking which packages have known update problems and not update them. For instance it would be good to get messages about scipy updates but scipy can't be auto updated. Or when Cryptography needed rust but we had no rust support.

**Comment by hoodmane** [comment]
Something like this that attempts to update Emscripten would be helpful too. Emscripten updates break more often than not, but it's good to have a program to nag us to look into it.

**Comment by ryanking13** [comment]
> A update script that would be in a GHA cron job that runs weekly and send a PR updating all packages if updates are available.

Yes, a cron job would be nice. We can adjust an interval if it is too short or long.

Note that I have some plans to remove pure python packages which can be installed directly from PyPI from the monorepo (#2580), which may reduce our burden to update packages in the future.

> Something like this that attempts to update Emscripten would be helpful too. 

Yes, indeed!

**Comment by ryanking13** [comment]
Thanks @ocefpaf!

**Pull request merged**